core/services/job/util.go

```diff
@@ -29,6 +28,15 @@ func EVMChainForJob(job *Job, set evm.ChainSet) (evm.Chain, error) {
 	return chain, nil
 }
 
+// EVMChainIDForJob parses the job spec and retrieves the evm chain id found
+func EVMChainIDForJobSpec(spec *OCR2OracleSpec) (chainID int64, ok bool) {
+	chainIDInterface, ok := spec.RelayConfig["chainID"]
+	if ok {
+		chainID, ok = chainIDInterface.(int64)
+	}
+	return
+}
```

Comment:
Would it help debugging to return an `error` for each so that we know if the field was missing or an unexpected type?